provider/sap/sap.go

```diff
@@ -0,0 +1,55 @@
+package sap
+
+import (
+	"fmt"
+	"github.com/messagebird/sachet"
+	"net/http"
+	"strings"
+	"time"
+)
+
+// Config is the configuration struct for Sap provider
+type Config struct {
+	URL      string `yaml:"url"`
+	AuthHash string `yaml:"auth_hash"`
+}
+
+// Sap contains the necessary values for the Sap provider
+type Sap struct {
+	Config
+}
+
+var sapHTTPClient = &http.Client{Timeout: time.Second * 20}
```

Comment:
`sapHTTPClient` doesn't need to be a package variable. You can instantiate it on the `NewSap` function and store in the Sap struct. 
Then on the Send `func` you can use it like `c.client.Do(request)`

Reply:
Done !